features/users.feature

```diff
@@ -14,6 +14,6 @@ Feature: Check users page/tab
         And I should see a "Active" link in the left menu
         And I should see a "Deactivated" link in the left menu
         And I should see a "All" link in the left menu
-        #And I should see a "admin" link in "td" "first-column "
-        And I should see 3 "admin" links
+        And I should see 2 "admin" links
```

Comment:
Good question why it did not break before, actually. But since I did find only two lower-case "admin" links and one "Admin" link also on 2.1, I decided for now to sync with that state. Maybe there should be third somewhere, but then I don't know... :)